scripts/Diffraction/isis_powder/osiris.py

```diff
@@ -172,13 +270,22 @@ def apply_drange_cropping(self, run_number_string, focused_ws):
 
     def get_vanadium_path(self, run_details):
         """
-        Get the vanadium path from the run details
+        Returns the vanadium path from the run details
         :param run_details: The run details of the run number
         :return: the vanadium path
         """
 
         return run_details.unsplined_vanadium_file_path
 
+    def get_vanadium_normalization_cutoff(self):
```

Comment:
Normally instrument settings are normally accessed directly rather than through getters (there are a handful of getters, so I accept it's a bit inconsistent). In this case I think you have done this so that you don't have to add the parameter for the other instruments (as this overrides an `abstract_inst` method that returns `None`). 
 However we can ask ourselves a couple of questions:

1.  Is this parameter ever likely to be changed (if not it can be hard-coded as it is in `calibrate`)?
https://github.com/mantidproject/mantid/blob/5096764bb676ca2be248be1fdcb690e0fbd1990a/scripts/Diffraction/isis_powder/routines/calibrate.py#L50-L56

2. If we do want to add a parameter, as `ReplaceSpecialValues` is called in other places, is it worth having the same threshold for all instances?
3. And in case of (2) perhaps this should be a parameter all instruments have?

Reply:
I added this parameter to deal with big data points caused by vanadium normalization. Yeah, I agree with you that it is called in multiple places and we need to make them consistent. This hard-coded ReplaceSpecialValues has been added recently by you. Why did you add it?